base/api.py

```
@@ -49,3 +90,45 @@ def get(self, request: Request, *args, **kwargs):
+
+        platform = request.query_params.get("platform")
+        handle = request.query_params.get("handle")
+
+        try:
+            reclaim_proof_func = async_to_sync(ReclaimProofRequest.init)
+            reclaim_proof_request = reclaim_proof_func(APP_ID, APP_SECRET, PROVIDER_ID, {"context": {"handle": handle}})
+            # reclaim_proof_request.set_app_callback_url("https://your-backend.com/receive-proofs")
+            reclaim_proof_request_config = reclaim_proof_request.to_json_string()
+
+            return JsonResponse({"reclaimProofRequestConfig": reclaim_proof_request_config})
+        except Exception as error:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)